modules/loop_configure_theme/loop_configure_theme.module

```diff
@@ -5,6 +5,26 @@
  * Add fields to a form, letting the administrator choose the header color and logo height.
  */
 function loop_configure_theme_form_system_theme_settings_alter(&$form, $form_state) {
+  $form['logout_settings'] = array(
```

Comment:
This should be moved to the Loop theme settings (themes/loop/theme-settings.php).